packages/react-native-bpk-component-calendar/README.md

```diff
@@ -11,10 +11,71 @@ npm install react-native-bpk-component-calendar --save-dev
 Because this package ships with native code, it is also necessary to add some native dependencies to your RN project:
 
 ### Android
-TODO - NEED TO WRITE THIS BIT
+
+Having installed the NPM package, add the following configuration to gradle:
+
+  1. Define the `react-native-bpk-component-calendar` project in your `settings.gradle` file:
+
+```groovy
+    include ':react-native-bpk-component-calendar'
+    project(':react-native-bpk-component-calendar').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-bpk-component-calendar/src/android')
+```
+
+  2. Add `react-native-bpk-component-calendar` as a dependency in your `build.gradle` file:
```

Comment:
Can mention which level `build.gradle`. Project or module. In this case it is module.